src/client/webpack.config.js

```diff
@@ -145,8 +161,9 @@ if (isProductionMode){
     }
     return loader;
   });
+
   webpackConfig.plugins.push(
-    new ExtractTextPlugin('[name].[contenthash].css'),
+    new ExtractTextPlugin('[name].css?[contenthash]'),
```

Comment:
Sounds sensible

On 21 April 2016 at 09:59, Dimitar Christoff <notifications@github.com>
wrote:

> In src/client/webpack.config.js
> <https://github.com/AdaptiveConsulting/ReactiveTraderCloud/pull/277#discussion_r60547016>
> :
>
> >    webpackConfig.plugins.push(
> > -    new ExtractTextPlugin('[name].[contenthash].css'),
> > +    new ExtractTextPlugin('[name].css?[contenthash]'),
>
> no but it follows the pattern for hash breaking that the .js files use as
> query arguments rather than part of the filename
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/AdaptiveConsulting/ReactiveTraderCloud/pull/277/files/bc76237b96903c6cdb6b76700a25e83c55c34c5b#r60547016>
>

-- 
 

*This email and any attachments to it may be confidential and are intended 
solely for the use of the individual to whom it is addressed. Any views or 
opinions expressed are solely those of the author and do not necessarily 
represent those of Adaptive Financial Consulting Ltd. *

*If you are not the intended recipient of this email, you must neither take 
any action based upon its contents, not copy nor show it to anyone.*

*Please contact the sender if you believe you have received this email in 
error.*

*Adaptive Financial Consulting Ltd is a company registered in England and 
Wales. Registered number: 08105955. Registered office: 2nd floor, 11 
Leadenhall Street, London EC3 V1LP.*